src/jira.js

```diff
@@ -46,7 +48,37 @@ export async function createIssue(summary, description) {
         },
         JSON.parse(settings.jiraFieldDefaults)
       )
-    });
+    };
+
+    const issue = await post("issue", data);
+
+    if (epic) {
+      try{
+        var data2 = {
+              "type": {
+                  "name": "Blocks"
+              },
+              "inwardIssue": {
+                  "key": issue.key
+              },
+              "outwardIssue": {
+                  "key": epic
+              },
+              "comment": {
+                  "body": "Linked related issue!"
+              }
+          };
+
+
+        var x = await post("issueLink", data2);
```

Comment:
if you're not doing anything with x, you can just have
`await post('issueLink', data2);`